projects/humans/c3d/protos/C3dGrf.proto

```diff
@@ -0,0 +1,30 @@
+#VRML_SIM R2019b utf8
+# tags: static, hidden
+
+PROTO C3dGrf [
```

Comment:
I would rename this proto `C3dGroundReactionForce` which is more understandable for the novice.